### DISCUSSION TITLE
Markdoc esbuild plugin

### DISCUSSION THREAD

**Discussion by toddw** [markdoc/markdoc#19] [Show and tell]
Markdoc looks awesome, I wanted to use it in a personal project but I didn't want to have to serve the content object from the backend. I made an esbuild plugin for markdown files that generates the content object and it works pretty well. This could be used as a template for how a webpack plugin could work too if someone wants to make one for that.

https://github.com/toddw/esbuild-markdoc-plugin

Feedback welcomed! 🚀

**Comment by mfix-stripe** [comment]
Super neat @toddw!

**Reply by mfix-stripe** [reply]
@toddw if you want, add `markdoc` to your repo's "Topics" for it to show up here: https://github.com/topics/markdoc

<img width="312" alt="image" src="https://user-images.githubusercontent.com/62121649/168187786-7c928b98-34fd-45db-a4b8-c47b4a68f0f6.png">

**Reply by toddw** [reply]
👍 added to topics, thank you!

**Comment by wobsoriano** [comment]
Nice!
